test/vstest.console.UnitTests/TestPlatformHelpers/TestRequestManagerTests.cs

```diff
@@ -965,8 +974,12 @@ public void RunTestsShouldCollectTelemetryForLegacySettings()
             this.testRequestManager.RunTests(payload, new Mock<ITestHostLauncher>().Object, new Mock<ITestRunEventsRegistrar>().Object, mockProtocolConfig);
 
             // Verify
-            Assert.IsTrue(actualRequestData.MetricsCollection.Metrics.TryGetValue(TelemetryDataConstants.LegacySettingElements, out var legacySettingsString));
-            StringAssert.Equals("Deployment, Scripts", legacySettingsString);
+            Assert.IsTrue(actualRequestData.MetricsCollection.Metrics.TryGetValue("VS.TestRun.LegacySettings.Elements", out var legacySettingsNodes));
+            StringAssert.Equals("Deployment,Scripts,Execution,AssemblyResolution,Timeouts,Hosts", legacySettingsNodes);
```

Comment:
Nit: Better if we have space after ',' ?  Like : "Deployment, Scripts, Execution.."

Reply:
Done